test/labeling/model/test_wrapper.py

```diff
@@ -0,0 +1,52 @@
+import unittest
+
+import numpy as np
+from sklearn.model_selection import GridSearchCV, RandomizedSearchCV
+
+from snorkel.labeling import SklearnLabelModel, create_cv_data
+
+
+class LabelModelWrapperTest(unittest.TestCase):
+    def _create_splits(self):
+        L_train = np.array(
+            [[1, 1, 1, 0], [0, 1, 1, 0], [1, 0, 0, 1], [0, 1, 0, 1], [0, 0, 0, 1]]
+        )
+        L_dev = np.array([[1, 1, 1, 0], [0, 1, 1, 0], [1, 0, 0, 1], [0, 1, 0, 1]])
+        Y_dev = np.array([1, 1, 0, 1])
+        L, Y, cv_split = create_cv_data(L_train, L_dev, Y_dev)
+        return L, Y, cv_split
+
+    def test_grid_search(self):
+        L, Y, cv_split = self._create_splits()
+        param_grid = [
+            {"lr": [0.01, 0.0000001], "l2": [0.001, 0.5], "metric": ["accuracy"]}
+        ]
+
+        clf = GridSearchCV(SklearnLabelModel(), param_grid, cv=cv_split)
+        clf.fit(L, Y)
+
+        self.assertGreaterEqual(clf.best_score_, 0.75)
+
+    def test_random_search(self):
```

Comment:
this probably isn't necessary, it'll just test sklearn param grid creation utilities